mantle/kola/tests/misc/files.go

```diff
@@ -198,6 +198,9 @@ func Denylist(c cluster.TestCluster) {
 		// DEL
 		"*\x7f*",
 	}
+	if runtime.GOARCH != "s390x" {
```

Comment:
Would be good to add a comment here on why we need this, and ideally some tracker issue to track dropping the dep again.